python/test/sentencepiece_test.py

```diff
@@ -24,11 +24,15 @@
 import os
 import pickle
 import unittest
+import threading
 import sentencepiece as spm
 
+HERE = os.path.dirname(os.path.abspath(__file__))
```

Comment:
This change looks right, to make it possible to run the tests from any directory rather than only in `python/`.

There are path-related changes that are not right though - after a test run with this branch, many `m_xxx.model` and m_xxx.vocab` files are left behind in `python/`. It'd be good to clean those up - or otherwise at a minimum extend the pattern for these files in the top-level `.gitignore`.